docs/dev/framework/content-elements.md

```diff
@@ -143,6 +143,47 @@ $GLOBALS['TL_LANG']['CTE']['my_content_element'] = [
 ```
 
 
+## Annotation
+
+{{< version "4.8" >}}
+
+Instead of tagging the content element controller service via the service configuration,
+the service tag can also be configured through annotations, as already used in the 
+code example above. The annotation can be used on the class of the content element 
+or on the method that will deliver the response.
+
+The following example sets the type of the content element to `my_example`, puts 
+it in the `media` category, sets the template name to `ce_some_example` and defines
+the renderer to be `inline` (which is the default):
```

Comment:
That's not exactly true, Contao ships with its own renderer (`forward`). I would either mention `esi` or none at all, as most users should not use that anyway 🙈

Reply:
Hmmm... I am confused now, in the core `inline` is used as well as `forward`.

Reply:
Not sure what you're referring to? The default renderer for Contao fragments is `forward`: https://github.com/contao/contao/blob/master/core-bundle/src/DependencyInjection/Compiler/RegisterFragmentsPass.php#L106

Reply:
In the FragmentHandler the default is `inline`:

https://github.com/contao/contao/blob/fd61bc4e34608f38112f97dfa1723337362ba229/core-bundle/src/Fragment/FragmentHandler.php#L66

But yeah, when registering a fragment controller, it would default to `forward` obviously. However, that means that the possible values are `forward`, `inline` and `esi`, correct?

Reply:
there's more than these three (like `hinclude` or any custom one). But these three are the only ones that make sense. Or maybe `inline` never makes sense now that we have `forward` 🤷‍♂